pkg/controller/podautoscaler/horizontal.go

```diff
@@ -579,8 +580,7 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 		rescaleReason = "Current number of replicas must be greater than 0"
 		desiredReplicas = 1
 	} else {
-
-		metricDesiredReplicas, metricName, metricStatuses, metricTimestamp, err = a.computeReplicasForMetrics(hpa, scale, hpa.Spec.Metrics)
+		metricDesiredReplicas, metricName, metricStatuses, _, err = a.computeReplicasForMetrics(hpa, scale, hpa.Spec.Metrics)
```

Comment:
you've removed the returned metricTimestamp from being considered here.

Reply:
The returned `metricTimestamp ` is not used in the function. So why we need it to be returned when it is not used?

Reply:
Ah, yeah, something got refactored away.  I suspect you should also be removing `timestamp` then.

Reply:
The `timestamp` variable is still used at line 593:
```go
klog.V(4).Infof("proposing %v desired replicas (based on %s from %s) for %s", metricDesiredReplicas, metricName, timestamp, reference)
```

Reply:
right, but it doesn't actually say anything useful, because it's not pulling a useful timestamp.  If we want that to be useful, it should be pulling the returned metric timestamp.

Reply:
Done. Now `metricTimestamp ` is being logged and `timestamp` variable is removed.